content/docs/intro/concepts/how-pulumi-works.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: "How Pulumi Works"
+title: "How Pulumi Deploys Infrastructure"
```

Comment:
This section doesn't fit quite as naturally in the new overall format of the "Architecture and Concepts" section (which is not really the "Concepts" section).  Two thoughts:
1. Should this be at the end of the TOC?
2. Should it be titled "Pulumi Internals" or "Pulumi Architecture" or similar?

Reply:
I like "Pulumi Architecture" and yes, I'll play around with putting this somewhere else to keep things clean and related.